swap/src/main.rs

```diff
@@ -297,6 +303,7 @@ async fn main() -> Result<()> {
                 bitcoin_network,
                 &wallet_data_dir,
                 monero_network,
+                seed,
```

Comment:
This is a symptom that maybe `init_wallet` is trying to do too much.

Reply:
Agreed. This needs some untangling that I am planning to tackle in cooldown: https://github.com/comit-network/xmr-btc-swap/issues/181